dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -516,7 +516,7 @@ def postgres_exclude(self) -> list[dict[str, str]]:
             return []
         return [
             # Exclude all combinations that are repeating python/postgres versions
-            {"python-version": python_version, "postgres-version": postgres_version}
+            {"python-version": python_version, "backend-version": postgres_version}
```

Comment:
That was the reason why we did not have postgres/mysql tests  in the first version.